packages/playground/src/App.js

```diff
@@ -185,14 +233,14 @@ function App() {
 
   const handleSendTypingIndicatorChange = useCallback(
     (e, checked) => {
-      setSendTypingIndicator(!!checked);
+      setSendTypingIndicator(checked);
     },
     [setSendTypingIndicator]
   );
 
   const handleUIDisabledChange = useCallback(
     (e, checked) => {
-      setDisabledUI(!!checked);
```

Comment:
why did we remove `!!`

Reply:
I already have falsy checks in state or session storage, plus I think it was confusing me during testing. Do you prefer keeping it? I can revert and I'll test to verify behavior if yes.